### PR TITLE
Feature/checkboxes and radio

### LINKED ISSUE #820: Checkbox Component

### LINKED ISSUE #821: Radio component

### PR DESCRIPTION
**Pull request by mjcarsjens**
Implementation for Radio Buttons and Checkboxes, reworked FormControlLabel and FormField to work with these.

**Closing issues**

- closes #820 
- closes #821

### CODE REVIEW

File: components/Checkbox/src/stories/react.stories.mdx [hunk 1]
```diff
@@ -31,331 +30,71 @@ import readme from "../../README.md";
...
-<FormControlLabel
-  label="Checkbox"
-  control={<Checkbox onChange={() => {}}></Checkbox>}
-></FormControlLabel>
-`,
-        },
-      },
-    }}
-  >
+  <Story name="Default" args={{}}>
```
Comment (bddjong) [inline components/Checkbox/src/stories/react.stories.mdx:33]: A few of the checkbox stories show a prop error in storybook

File: components/Checkbox/src/stories/bem.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,198 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS/Input/Checkbox"
```
Comment (bddjong) [inline components/Checkbox/src/stories/bem.stories.mdx:8]: Can we have BEM stories for checkbox and radio groups?

**Review by bddjong — CHANGES_REQUESTED** [review]

**Comment by bddjong** [comment]
Keyboard navigation on the radio button form field is broken on chrome. It’s not selecting the radio button which has a hover

File: components/Checkbox/src/stories/react.stories.mdx (hunk 1 above)
Comment (mjcarsjens) [reply components/Checkbox/src/stories/react.stories.mdx:33]: Works on my machine ¯\_(ツ)_/¯

File: components/Checkbox/src/index.scss [hunk 3]
```diff
@@ -1,108 +1,118 @@
...
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: var(--utrecht-custom-checkbox-size);
+  aspect-ratio: 1;
+  background-color: var(--utrecht-custom-checkbox-background-color);
+  border-width: var(--utrecht-custom-checkbox-border-width);
+  border-style: solid;
+  border-color: var(--utrecht-custom-checkbox-border-color);
+  border-radius: var(--utrecht-custom-checkbox-border-radius);
```
Comment (bddjong) [inline components/Checkbox/src/index.scss:33]: I think the `utrecht-form-input` border is identical. Maybe it's useful to add those tokens as fallback values so other organisations won't have to define as many tokens.

File: components/Checkbox/src/index.scss [hunk 4]
```diff
@@ -1,108 +1,118 @@
...
+  font-size: var(--utrecht-custom-checkbox-icon-size);
+  color: var(--utrecht-custom-checkbox-checked-color);
 }
 
-.denhaag-checkbox--error:hover + .denhaag-form-control-label__label,
-.denhaag-checkbox--error.denhaag-checkbox--hover + .denhaag-form-control-label__label {
-  color: var(--denhaag-checkbox-error-hover-color);
+.denhaag-checkbox.denhaag-checkbox--error .denhaag-checkbox__icon {
+  border-width: var(--utrecht-custom-checkbox-invalid-border-width);
+  border-color: var(--utrecht-custom-checkbox-invalid-border-color);
```
Comment (bddjong) [inline components/Checkbox/src/index.scss:60]: Same here and for other styles if they already exist as `form-input` tokens. (Mind that we also have a few `denhaag-input` tokens that could be used as fallbacks)

**Review by bddjong — APPROVED** [review]

**Pull request merged**